blocks/plain-text/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import TextareaAutosize from 'react-autosize-textarea';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function PlainText( { onChange, className, ...props } ) {
+	return (
+		<TextareaAutosize
+			useCacheForDOMMeasurements
```

Comment:
`useCacheForDOMMeasurements` - can you explain what this thing does? We didn't have it before.

Reply:
I don't know exactly :), I was just playing with `TextareaAutosize` and found this on their docs. this was just a leftover, I removed it.